src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+function sortRowsByNumber(rows, childSelector) {
+  let result = [];
+  const childNomer = childSelector + 1;
+
+  if (childSelector === 0 || childSelector === 1) {
```

Comment:
The condition 'childSelector === 0 || childSelector === 1' seems to assume that only the first two columns contain text data. Ensure this logic aligns with your table structure.